src/Instrumentation/Laravel/src/Watchers/LogWatcher.php

```diff
@@ -59,7 +60,7 @@ public function recordLog(MessageLogged $log): void
 
         $record = (new LogRecord($log->message))
             ->setSeverityText($log->level)
-            ->setSeverityNumber(Psr3::severityNumber($log->level))
+            ->setSeverityNumber(Severity::fromPsr3($log->level))
```

Comment:
@ChrisLightfootWild I think this requires API 1.1.0 and therefore dropping 8.0 support - how does that sound?